dev-python/pycpio/pycpio-1.3.1.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{11..12} )
+inherit distutils-r1
+distutils_enable_tests unittest
```

Comment:
We usually put this line directly after the dependencies because in the eclass this function modifies the dependencies.

Reply:
I think i just fixed that